### ISSUE TITLE
Reno doesn't handle long json strings well in notifications

### ISSUE THREAD

**Issue by LostKobrakai** [ryancramerdesign/ProcessWire#1429]
2.6.17 with System Notifications and the setting to let notices be notices.

Had also an issue, where I could toggle the notice, but the opened second line wasn't wrapping because the json didn't consist whitespaces. Maybe json could in such cases also get a pretty-print function like array's now have.

![screenshot 2015-10-03 14 53 53](https://cloud.githubusercontent.com/assets/2489835/10262996/b09aa422-69de-11e5-8ee6-2bc699a70504.png)

cc @Renobird

**Comment by ryancramerdesign** [comment]
In trying to see the issue you mentioned, I noticed that the "leave them alone" setting for the placement option was causing Notices to not be shown at all, so I have fixed that. 

As for long JSON strings, it sounds like the string literally had no whitespace in it somehow (spaces replaced with &nbsp; ?). It seems like this situation would only occur in debugging scenarios, so I'm okay with less than perfect output there. 

As for outputting JSON strings your screenshot looks like it was a PHP array run through print_r()? When it comes to arrays, it's easy enough for Notices to detect that because it's a PHP type. But a JSON string is just another string, and that would lead to us having to get into JSON detection in the Notice, and how many people are outputting JSON strings in Notices anyway? :) What'd I'd recommend is this:
`````
wire()->message(json_decode($jsonString, true)); 
`````
I'm thinking that may give you the output you are looking for.
